ui/src/main/java/edu/wpi/grip/ui/MainWindowController.java

```diff
@@ -60,6 +64,7 @@
 /**
  * The Controller for the application window.
  */
+@SuppressWarnings("PMD.GodClass") // temporary
```

Comment:
Add an issue to track this or make sure that this isn't in the final version you want to merge.